### ISSUE TITLE
BulkIndexDescriptor IfPrimaryTerm sets IfSequenceNumber

### ISSUE THREAD

**Issue by takridge** [elastic/elasticsearch-net#4078]
**NEST/Elasticsearch.Net version**: 7.3

**Elasticsearch version**: 7.3

**Description of the problem including expected versus actual behavior**: When utilizing the .IndexMany() of .Bulk() in the ElasticClient, calling the .IfPrimaryTerm() does not set the IfPrimaryTerm ("if_primary_term") DataMember, but rather the IfSequenceNumber ("if_seq_no") DataMember.

**Steps to reproduce**:
1. Already have index created
2. Instantiate ElasticClient connected to existing cluster
3. Call `.Bulk(x => x.IndexMany(documents, (d, doc) => d.IndexName("index_name").IfSequenceNumber(doc.SequenceNumber).IfPrimaryTerm(doc.PrimaryTerm)))`
4. Receive error from server: `Validation Failed: 1: ifSeqNo is set, but primary term is [0];...`

**Provide `DebugInformation`**:
```
...
{"index":{"_id":"anExampleId","_index":"index_name","if_seq_no":1335}}
...
```

**Relevant Nest Code**:
master: src\Nest\Document\Multiple\Bulk\BulkOperation\BulkIndex.cs:86

`public BulkIndexDescriptor<T> IfPrimaryTerm(long? primaryTerm) => Assign(primaryTerm, (a, v) => a.IfSequenceNumber = v);`

**Comment by takridge** [comment]
This also affects `BulkUpdateDescriptor` in src\Nest\Document\Multiple\Bulk\BulkOperation\BulkUpdate.cs:265-266

```
        public BulkUpdateDescriptor<TDocument, TPartialDocument> IfPrimaryTerm(long? primaryTerm) =>
            Assign(primaryTerm, (a, v) => a.IfSequenceNumber = v);
```

should be

```
        public BulkUpdateDescriptor<TDocument, TPartialDocument> IfPrimaryTerm(long? primaryTerm) =>
            Assign(primaryTerm, (a, v) => a.IfPrimaryTerm = v);
```

**Comment by russcam** [comment]
Closing this as it's fixed in #4080 which has now been merged. It'll be in the next release of 6.x and 7.x
